exporting/tests/exporting_fixtures.c

```diff
@@ -127,3 +127,35 @@ int teardown_initialized_engine(void **state)
 
     return 0;
 }
+
+int setup_prometheus(void **state)
+{
+    (void)state;
+
+    prometheus_exporter_instance = calloc(1, sizeof(struct instance));
+
+    setup_rrdhost();
+
+    prometheus_exporter_instance->config.update_every = 10;
+
+    prometheus_exporter_instance->config.options |=
+        EXPORTING_OPTION_SEND_NAMES | EXPORTING_OPTION_SEND_CONFIGURED_LABELS | EXPORTING_OPTION_SEND_AUTOMATIC_LABELS;
```

Comment:
Here we have more notorious what I said above, during the tests you are sending both `configured labels` and `automatic labels`.

Reply:
This is just to check that both are working correctly.